README.md

```diff
@@ -1,110 +1,31 @@
 ![](https://raw.githubusercontent.com/auth0/wp-auth0/master/banner-1544x500.png)
 
-Wordpress Plugin for Auth0
+Login by Auth0
 ====
 
 Single Sign On for Enterprises + Social Login + User/Passwords. For all your WordPress instances. Powered by Auth0.
 
-Demo: <http://auth0wp.azurewebsites.net>
-
-Documentation: <https://auth0.com/docs/cms>
+Download and install <https://wordpress.org/plugins/auth0/>
+Documentation: <https://auth0.com/docs/cms/wordpress>
 
 ## Important note on 3.5
 
 This is a major update that requires changes to your Auth0 Dashboard to be completed. You can save a new [API token](https://auth0.com/docs/api/management/v2/tokens#get-a-token-manually) in your Basic settings in wp-admin before upgrading and the changes will be made automatically during the update. Otherwise, please review your [Client Advanced Settings](https://auth0.com/docs/cms/wordpress/configuration#client-setup), specifically your Grant Types, and [authorize your Client for the Management API](https://auth0.com/docs/cms/wordpress/configuration#authorize-the-client-for-the-management-api). 
 
-## Contributions
-
-All PR should be done towards the `dev` branch.
-
 ## Installation
 
-Before you start, **make sure the admin user has a valid email that you own**, read the Technical Notes for more information.
-
-1. Install from the  **WordPress** Store or upload the entire wp-auth0 folder to the /wp-content/plugins/ directory.
-2. Activate the plugin through the 'Plugins' menu in  **WordPress**.
-3. Create an account in Auth0 (https://auth0.com) and add a new Application.
-4. Copy the Client ID, Client Secret and Domain from the Settings of the Application.
-5. On the Settings of the Auth0 application change the Callback URL to be: http://your-domain/index.php?auth0=1. Using TLS/SSL is recommended for production.
-6. Go back to  **WordPress** Settings - Auth0 Settings edit the Domain, Client ID and Client Secret with the ones you copied from Auth0 Dashboard.
-
-## Implicit Flow
-
-There are cases where the server is behind a firewall and does not have access to internet (or at least, can't reach the Auth0 servers). In those cases, you can enable the Auth0 Implicit Flow in the advanced settings of the Auth0 Settings page.
+[Please see the Installation page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/installation)
```

Comment:
Does this also mention that you should set your client type in dashboard to SPA?

Reply:
@cocojoe - I'm removing the reference to Implicit here, not related to the added line below